src/window/building/common.c

```diff
@@ -83,7 +103,8 @@ void window_building_draw_employment(building_info_context *c, int y_offset)
             c->x_offset + 70 + width, y_offset + 10, FONT_SMALL_BLACK);
         lang_text_draw(69, 0, c->x_offset + 70 + width, y_offset + 10, FONT_SMALL_BLACK);
         lang_text_draw(69, text_id, c->x_offset + 70, y_offset + 26, FONT_SMALL_BLACK);
-    } else {
+    }
+    else {
```

Comment:
Formatting: } and else on the same line please.

Reply:
done